packages/openactive-integration-tests/test/features/payment/payment-reconciliation-detail-validation/implemented/payment-reconciliation-detail-validation-invalid-paymentProviderId-test.js

```diff
@@ -0,0 +1,14 @@
+const { FeatureHelper } = require('../../../../helpers/feature-helper');
+const { invalidDetailsTest } = require('../common');
+
+FeatureHelper.describeFeature(module, {
+  testCategory: 'payment',
+  testFeature: 'payment-reconciliation-detail-validation',
+  testFeatureImplemented: true,
+  testIdentifier: 'payment-reconciliation-detail-validation-invalid-paymentProviderId',
+  testName: 'Payment reconciliation detail validation - invalid paymentProviderId',
+  testDescription: 'B should return an InvalidPaymentDetailsError due to invalid requisition data',
+  testOpportunityCriteria: 'TestOpportunityBookablePaid',
+  controlOpportunityCriteria: 'TestOpportunityBookable',
+},
+invalidDetailsTest('invalidPaymentProviderId'));
```

Comment:
This abstraction is not necessary, and `invalidReconciliationDetails` should be used instead of both of these